services/api/src/clients/harborClient.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+const got = require('got');
+const R = require('ramda');
+
+const { HARBOR_ADMIN_PASSWORD } = process.env;
+
+const lagoonHarborRoute = R.compose(
+  R.defaultTo('http://172.17.0.1:8084'),
+  R.find(R.test(/harbor-nginx/)),
+  R.split(','),
+  R.propOr('', 'LAGOON_ROUTES'),
+)(process.env);
+
+// Test to see if we have a local harbor or not
+// curl 172.17.0.1:8084; if 200: use that. if not, use harbor-nginx?
```

Comment:
I don't see such a test?

Reply:
This was a comment from back when I was still trying to figure out the logic for making this work. Since we've changed the way this all works since then, this is no longer relevant.